warn/warn_test.go

```diff
@@ -1235,3 +1235,68 @@ def foobar():
     fail()
 `, []string{}, scopeEverywhere)
 }
+
+func TestUnreachableStatementWarning(t *testing.T) {
+	// after return
+	checkFindings(t, "unreachable", `
+def foo():
+  return
+  bar()
+  baz()
+`, []string{
+		`:3: The statement is unreachable.`,
+	}, scopeEverywhere)
+
+	// after fail()
+	checkFindings(t, "unreachable", `
+def foo():
+  fail("die")
+  bar()
+  baz()
+`, []string{
+		`:3: The statement is unreachable.`,
+	}, scopeEverywhere)
+
+	// after break and continue
+	checkFindings(t, "unreachable", `
+def foo():
+  for x in y:
+    if x:
+      break
+      bar()
+    if y:
+      continue
+      bar()
+
+def bar():
+  for x in y:
+    if x:
+      break
+    elif y:
+      continue
+    else:
+      return x
+
+    foo()
```

Comment:
Add a comment on unreachable lines (it's not obvious which one is line 19 otherwise)

Reply:
Done.